### ISSUE TITLE
Show line/column numbers in output

### ISSUE THREAD

**Issue by pelotom** [nadeesha/ts-prune#35]
It would be cool if exports had line numbers so that e.g. in [iTerm 2](https://www.iterm2.com/) you can command-click on such a path to open it to the exact location of the export in the file:

```
Foo @ ./src/some/path/Foo.ts:42
```

Even better would be line plus column number:

```
Foo @ ./src/some/path/Foo.ts:42:7
```

**Comment by nadeesha** [comment]
@pelotom Relative paths are implemented with `v0.2.3`. Line counts maybe a bit trickier. I'll see what I can do about it, possibly later.

**Comment by pelotom** [comment]
Awesome, thanks! I updated the issue title and description accordingly.
